### ISSUE TITLE
Lazy loading finished notification

### ISSUE THREAD

**Issue by julianobrasil** [ngrx/platform#2394]
I'm not sure if this is a fair thing to ask for. After updating my projects to angular 9 and the latest version of NgRx, I introduced a behavior that was inexistent before. Basically, in one of my lazy loaded modules, I was accessing, in the constructor of the module declaration class, the store declared on that same module (using `forFeature`). I didn't have problems with that in the previous versions. Now, part of my app stopped working and, after a quick look, the problem was that the lazy part of the store wasn't ready to be used yet in the constructor.

## Describe any alternatives/workarounds you're currently using

With a `setTimeout` I was able to fix the problem. But, isn't it the case there should be something like `onFeatureStoreReady`?

## Other information:

```
@angular-devkit/architect    0.900.1
@angular-devkit/core         9.0.1
@angular-devkit/schematics   9.0.1
@schematics/angular          9.0.1
@schematics/update           0.900.1
rxjs                         6.5.3
nx                           8.12.2
@ngrx/store                  8.6.0
```

## If accepted, I would be willing to submit a PR for this feature

[ ] Yes (Assistance is provided if you need help submitting a pull request)
[x] No
But I'm not discarding the possibility of helping with that. It just would take some time as I should learn a little bit about the codebase.

**Comment by brandonroberts** [comment]
@julianobrasil can you provide a small reproduction of the issue?

**Comment by julianobrasil** [comment]
I'll work on that.

**Comment by julianobrasil** [comment]
@brandonroberts, I've built up something on stackblitz (https://stackblitz.com/edit/angular-ngrx-issue-repro), but it worked smoothly, without the need for the timeout. So it's apparently not a general problem of `@ngrx`. It's really happening in 2 of my projects though (one occurrence in each project). Maybe it's something related to a more complex structure than the one I built on the stackblitz repro. (I even tried to add two levels of lazy loading, but with no problem at all).

For now, I'm gonna close this one until I can eventually reproduce it properly.
